agenthub/monologue_agent/agent.py

```diff
@@ -71,6 +72,10 @@
     'BROWSE google.com',
     '<form><input type="text"></input><button type="submit"></button></form>',
     'I can browse the web too!',
+    'If I have done some work and I want to push it to github, I can do that also!',
+    "Let's do it.",
+    'PUSH owner/repo branch',
+    'The repo was successfully pushed to https://github.com/owner/repo/branch',
```

Comment:
Just a small question about this last thought, will it be saved to memory? It seems it will get on line 178 (of this branch) as a NullObservation, since output_type is PUSH and we don't have an ObservationType.PUSH in the ifs there. But I can't test at the moment, and I might be missing something.

Related, this line reads to me as it was a thought of an Action.Think, could it be worth to save it instead as an AgentMessageObservation? I'm not sure we have yet any AgentMessageObservation in use by monologue agent, it... might be time to teach it they exist 😄

Reply:
Ah, good catch, thank you! I updated this.